multi_task_example.ipynb

```diff
@@ -199,7 +203,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "max_epochs = 200 if not os.getenv(\"CI\", False) else 2"
+    "max_epochs = 10 if not os.getenv(\"CI\", False) else 2"
```

Comment:
`max_epochs = 200` outside CI seems better

Reply:
I forgot to restore it, thks

Reply:
done